tests/notebooks/training/training-integration.ipynb

```diff
@@ -411,7 +417,7 @@
    "source": [
     "PYTORCHJOB_NAME = \"pytorch-dist-mnist-gloo\"\n",
     "PYTORCHJOB_CONTAINER = \"pytorch\"\n",
-    "PYTORCHJOB_IMAGE = \"gcr.io/kubeflow-ci/pytorch-dist-mnist-test:v1.0\""
+    "PYTORCHJOB_IMAGE = \"kubeflow/pytorch-dist-mnist:v1-3a360ba\""
```

Comment:
I understand that pinning to this tag instead of using latest will help us prevent potential issues, but I also see this will be adding a chore and in the future we may face the same issue as this PR is trying to close if the image is outdated.

I think we should at leas write a comment that says this image works with training operator 1.7 (maybe 1.8 as well) so we know it has to be updated on every release(?) of the training operator.

Reply:
Added a comment for updating the image on new releases with the latest in the registry.

Reply:
BTW ... that new image is 10x larger than the previous one (~1G vs ~10G) ... I believe it has the full CUDA thing in it ... not all environments will need it ... and some might fail getting resource exhaustion while pulling the image.

Reply:
Hey @motjuste, thanks for raising this. I filed an issue about it https://github.com/canonical/charmed-kubeflow-uats/issues/69, including context from our live discussion as well. I'll go ahead and merge this PR in order to fix the test for now, and we 'll look for tackling this in the future, since right now, we are pressed from the release